server/database/queries/cohort/putSpecificCohort.js

```diff
@@ -0,0 +1,11 @@
+const connection = require('../../config/connection');
+
+const putSpecificCohort = (req) => {
+  const { name, description, imgUrl, githubLink, cohortId } = req;
+  return connection.query(
+    'UPDATE cohort SET name = $1, description = $2, img_url = $3, github_link = $4 WHERE id = $5 ',
```

Comment:
@Mu7ammadAbed My comment [here](https://github.com/GSG-G8/ca-wiki/pull/47#discussion_r400924612) might be helpful your endpoint - you can design PUT endpoints to accept any number of arguments, instead of always expecting every argument to be provided by the frontend.

Reply:
Here's an example of the query syntax: https://stackoverflow.com/a/19358694
You can use a question mark in the query and then pass an object in the arguments array to match with the question mark.  In your code, I think that means it could look like this:
`return connection.query('UPDATE cohort SET ? WHERE id = ?', [req, req.cohortId]);`

Once you do that, you should be able PUT any number / combination of the arguments (for example, only `name` or only `img_url` or both `name` and `description` together)

Hopefully that works - and if not, sorry for sending you on the wrong path.  This change isn't incredibly important for this project, but maybe it would help for a future project.

Reply:
Nice! this is very helpful